tests/rptest/scale_tests/tiered_storage_cache_stress_test.py

```diff
@@ -267,3 +282,158 @@ def streaming_cache_test(self, limit_mode, log_segment_size):
 
         # At least one node should have _something_ in its cache, or something is wrong with our test
         assert any(nodes_cache_used) is True
+
+    @cluster(num_nodes=4)
+    def tiny_cache_test(self):
+        """
```

Comment:
nit: just thinking about reducing complexity of the test, given there's already a lot in here, maybe consider just running on one node with RF=1? I know it goes against being end-to-end, but these tests seem pretty targeted, and I don't imagine we'd lose much coverage by not having followers

Don't feel too strongly about it though